packages/manager/modules/pci/src/projects/new/payment/components/add/controller.js

```diff
@@ -42,6 +42,13 @@ export default class PciProjectNewPaymentMethodAddCtrl {
     );
   }
 
+  getPayPalChargeAmount() {
+    const uCent = 1000000; // micro cent factor -> 10^-6
+    const priceInCent = this.voucherPlan.pricings[0].price / uCent;
```

Comment:
do you have cases where pricings could contains 0 values ? or more then 1 ?

Reply:
Ah, got your point.
I think no, but i will ask Integration to be sur.

Reply:
So no, can't be 0.

Reply:
@mohammed-zahaf ok it can't have 0 values, but what about more than one?

Reply:
Fixed.